openbb_platform/extensions/quantitative/openbb_quantitative/quantitative_router.py

```diff
@@ -415,8 +432,10 @@ def sortino_ratio(
 
 
 @router.command(methods=["POST"])
-def skewness(data: List[Data], target: str, window: PositiveInt) -> OBBject[List[Data]]:
-    """Get Skewness.
+def skewness(
+    data: List[Data], target: str, window: PositiveInt, index: str = "date"
```

Comment:
Sharpe and Kurtosis have a default state for `window`, so should there be one here and Quantile?

Reply:
added